### PR TITLE
Task# 7038 edit-product

### PR DESCRIPTION
**Pull request by dinhmuoi97**
## Redmine URL
 - https://edu-redmine.framgia.vn/issues/7038

## Reproduce steps
 - for Bug only.

## What kind of page or function is need to check
 - 
 - 
 - 

## Before｜After
Before|After|
--|--|
Before content|After content|

## Check contents list
 - 

## Other
 - Test ENV
 - Batch, services,... need run for this pull.
peek 2019-03-04 22-48

### CODE REVIEW

**Comment by dinhmuoi97** [comment]
ready

**Comment by dinhmuoi97** [comment]
ready

File: app/controllers/products_controller.rb [hunk 1]
```diff
@@ -51,4 +66,8 @@ def product_params
   def admin_user
     redirect_to root_url unless current_user.admin?
   end
+
+  def search_product
+    @product = Product.find_by id: params[:id]
```
Comment (yanghoxom) [inline app/controllers/products_controller.rb]: Xử lý khi không find đc

File: app/views/products/index.html.erb [hunk 2]
```diff
@@ -50,6 +50,7 @@
                 <td><%= y.sale %></td>
                 <td>
                   <%= link_to t("xoa"), y, method: :delete, class: "btn btn-primary"%>
+                  <%= link_to t("sua"), edit_product_path(y), class: "btn btn-danger" %>
```
Comment (yanghoxom) [inline app/views/products/index.html.erb]: Dùng tiếng lào đi

**Comment by dinhmuoi97** [comment]
ready

**Pull request merged**